### ISSUE TITLE
Update cluster is creating EBS volumes that exist

### ISSUE THREAD

**Issue by chrislovecnm** [kubernetes/kops#1339]
Created a private cluster that is a single master.  This by default does not create a bastion.  I added a new instance group as a bastion.  Below is the cluster update:

```console
$ kops update cluster
Using cluster from kubectl context: liea-ginny.dev.datapipe.io

I0103 20:31:42.335306   49764 populate_cluster_spec.go:243] Defaulting DNS zone to: Z2PKVP3BDFE581
W0103 20:31:42.486059   49764 apply_cluster.go:207] Using nodeup location from NODEUP_URL env var: "https://kops-devel.s3.amazonaws.com/kops/git-90d4097/linux/amd64/nodeup"
I0103 20:31:43.062506   49764 executor.go:91] Tasks: 0 done / 80 total; 30 can run
I0103 20:31:45.321521   49764 executor.go:91] Tasks: 30 done / 80 total; 17 can run
I0103 20:31:46.368975   49764 executor.go:91] Tasks: 47 done / 80 total; 24 can run
I0103 20:31:47.979243   49764 executor.go:91] Tasks: 71 done / 80 total; 6 can run
I0103 20:31:48.268890   49764 dnsname.go:102] AliasTarget for "bastion-liea-ginny.dev.datapipe.io." is "bastion-liea-ginny-627324423.eu-west-1.elb.amazonaws.com."
I0103 20:31:48.353083   49764 dnsname.go:102] AliasTarget for "api.liea-ginny.dev.datapipe.io." is "api-liea-ginny-1827685807.eu-west-1.elb.amazonaws.com."
W0103 20:31:48.451638   49764 dnsname.go:110] Unable to find load balancer with DNS name: "bastion-liea-ginny-627324423.eu-west-1.elb.amazonaws.com."
I0103 20:31:48.626365   49764 executor.go:91] Tasks: 77 done / 80 total; 3 can run
I0103 20:31:48.845834   49764 executor.go:91] Tasks: 80 done / 80 total; 0 can run
Will create resources:
  EBSVolume/eu-west-1a.etcd-main.liea-ginny.dev.datapipe.io
  	AvailabilityZone    	eu-west-1a
  	VolumeType          	gp2
  	SizeGB              	20
  	Encrypted           	false
  	Tags                	{k8s.io/etcd/main: eu-west-1a/eu-west-1a, k8s.io/role/master: 1, Name: eu-west-1a.etcd-main.liea-ginny.dev.datapipe.io, KubernetesCluster: liea-ginny.dev.datapipe.io}

  EBSVolume/eu-west-1a.etcd-events.liea-ginny.dev.datapipe.io
  	AvailabilityZone    	eu-west-1a
  	VolumeType          	gp2
  	SizeGB              	20
  	Encrypted           	false
  	Tags                	{k8s.io/etcd/events: eu-west-1a/eu-west-1a, k8s.io/role/master: 1, Name: eu-west-1a.etcd-events.liea-ginny.dev.datapipe.io, KubernetesCluster: liea-ginny.dev.datapipe.io}

  SecurityGroup/bastion-elb.liea-ginny.dev.datapipe.io
  	Description         	Security group for bastion ELB
  	VPC                 	name:liea-ginny.dev.datapipe.io id:vpc-799cc01d
  	RemoveExtraRules    	[port=22]

  SecurityGroup/bastion.liea-ginny.dev.datapipe.io
  	Description         	Security group for bastion
  	VPC                 	name:liea-ginny.dev.datapipe.io id:vpc-799cc01d
  	RemoveExtraRules    	[port=22]

  SecurityGroupRule/bastion-egress
  	SecurityGroup       	name:bastion.liea-ginny.dev.datapipe.io
  	CIDR                	0.0.0.0/0
  	Egress              	true

  SecurityGroupRule/ssh-elb-to-bastion
  	SecurityGroup       	name:bastion.liea-ginny.dev.datapipe.io
  	Protocol            	tcp
  	FromPort            	22
  	ToPort              	22
  	SourceGroup         	name:bastion-elb.liea-ginny.dev.datapipe.io

  SecurityGroupRule/bastion-elb-egress
  	SecurityGroup       	name:bastion-elb.liea-ginny.dev.datapipe.io
  	CIDR                	0.0.0.0/0
  	Egress              	true

  SecurityGroupRule/ssh-external-to-bastion-elb-0.0.0.0/0
  	SecurityGroup       	name:bastion-elb.liea-ginny.dev.datapipe.io
  	CIDR                	0.0.0.0/0
  	Protocol            	tcp
  	FromPort            	22
  	ToPort              	22

  SecurityGroupRule/bastion-to-master-ssh
  	SecurityGroup       	name:masters.liea-ginny.dev.datapipe.io id:sg-14b4cb72
  	Protocol            	tcp
  	FromPort            	22
  	ToPort              	22
  	SourceGroup         	name:bastion.liea-ginny.dev.datapipe.io

  SecurityGroupRule/bastion-to-node-ssh
  	SecurityGroup       	name:nodes.liea-ginny.dev.datapipe.io id:sg-15b4cb73
  	Protocol            	tcp
  	FromPort            	22
  	ToPort              	22
  	SourceGroup         	name:bastion.liea-ginny.dev.datapipe.io

  LaunchConfiguration/bastion.liea-ginny.dev.datapipe.io
  	ImageID             	kope.io/k8s-1.4-debian-jessie-amd64-hvm-ebs-2016-10-21
  	InstanceType        	t2.medium
  	SSHKey              	name:kubernetes.liea-ginny.dev.datapipe.io-8b:82:20:c2:d3:ed:b0:1e:a2:5a:ce:47:28:b0:87:67 id:kubernetes.liea-ginny.dev.datapipe.io-8b:82:20:c2:d3:ed:b0:1e:a2:5a:ce:47:28:b0:87:67
  	SecurityGroups      	[name:bastion.liea-ginny.dev.datapipe.io]
  	AssociatePublicIP   	false
  	IAMInstanceProfile  	name:bastions.liea-ginny.dev.datapipe.io id:bastions.liea-ginny.dev.datapipe.io
  	RootVolumeSize      	20
  	RootVolumeType      	gp2

  LoadBalancer/bastion.liea-ginny.dev.datapipe.io
  	ID                  	bastion-liea-ginny
  	Subnets             	[name:utility-eu-west-1a.liea-ginny.dev.datapipe.io id:subnet-f2600aaa]
  	SecurityGroups      	[name:bastion-elb.liea-ginny.dev.datapipe.io]
  	Listeners           	{22: {"InstancePort":22}}
  	HealthCheck         	{"Target":"TCP:22","HealthyThreshold":2,"UnhealthyThreshold":2,"Interval":10,"Timeout":5}
  	ConnectionSettings  	{"IdleTimeout":300}

  AutoscalingGroup/bastion.liea-ginny.dev.datapipe.io
  	MinSize             	1
  	MaxSize             	1
  	Subnets             	[name:eu-west-1a.liea-ginny.dev.datapipe.io id:subnet-f1600aa9]
  	Tags                	{KubernetesCluster: liea-ginny.dev.datapipe.io, k8s.io/role/bastion: 1, Name: bastion.liea-ginny.dev.datapipe.io}
  	LaunchConfiguration 	name:bastion.liea-ginny.dev.datapipe.io

  LoadBalancerAttachment/bastion-elb-attachment
  	LoadBalancer        	name:bastion.liea-ginny.dev.datapipe.io id:bastion-liea-ginny
  	AutoscalingGroup    	name:bastion.liea-ginny.dev.datapipe.io id:bastion.liea-ginny.dev.datapipe.io

Will modify resources:
  DNSName/bastion-liea-ginny.dev.datapipe.io
  	TargetLoadBalancer  	 <nil> -> name:bastion.liea-ginny.dev.datapipe.io id:bastion-liea-ginny

Must specify --yes to apply changes
```

Specifically the EBS volumes are concerning

```

  EBSVolume/eu-west-1a.etcd-main.liea-ginny.dev.datapipe.io
  	AvailabilityZone    	eu-west-1a
  	VolumeType          	gp2
  	SizeGB              	20
  	Encrypted           	false
  	Tags                	{k8s.io/etcd/main: eu-west-1a/eu-west-1a, k8s.io/role/master: 1, Name: eu-west-1a.etcd-main.liea-ginny.dev.datapipe.io, KubernetesCluster: liea-ginny.dev.datapipe.io}

  EBSVolume/eu-west-1a.etcd-events.liea-ginny.dev.datapipe.io
  	AvailabilityZone    	eu-west-1a
  	VolumeType          	gp2
  	SizeGB              	20
  	Encrypted           	false
  	Tags                	{k8s.io/etcd/events: eu-west-1a/eu-west-1a, k8s.io/role/master: 1, Name: eu-west-1a.etcd-events.liea-ginny.dev.datapipe.io, KubernetesCluster: liea-ginny.dev.datapipe.io}
```

**Comment by chrislovecnm** [comment]
This is probably because the volumes are not mounted

**Comment by justinsb** [comment]
I think this probably happened when you create a cluster with `kops create cluster --target terraform`, and should have been fixed by #1421 ?  In any case, I believe fixed in 1.5.0-alpha2?

**Comment by justinsb** [comment]
Closing as saw this with terraform output and fixed in early alphas.  Please reopen if you hit again.
